python/tests/conftest.py

```diff
@@ -56,13 +56,19 @@
 from arcticdb_ext.storage import Library
 from azure.storage.blob import BlobServiceClient, generate_account_sas, ResourceTypes, AccountSasPermissions
 
-
 from tests.util.mark import (
     AZURE_TESTS_MARK,
     MONGO_TESTS_MARK,
     REAL_S3_TESTS_MARK,
 )
 
+import hypothesis as h
```

Comment:
I don't see any value in aliasing `hypothesis` to `h`. This is not a commonly used alias and will just cause confusion.